rocky/rocky/settings.py

```diff
@@ -181,59 +176,58 @@
 AUTH_USER_MODEL = "account.KATUser"
 
 # Database
-# https://docs.djangoproject.com/en/3.2/ref/settings/#databases
-POSTGRES_USER = os.getenv("ROCKY_DB_USER")
-POSTGRES_PASSWORD = os.getenv("ROCKY_DB_PASSWORD")
-POSTGRES_DB = os.getenv("ROCKY_DB")
-POSTGRES_DB_HOST = os.getenv("ROCKY_DB_HOST")
-POSTGRES_DB_PORT = os.getenv("ROCKY_DB_PORT")
-
-DATABASES = {
-    "default": {
+# https://docs.djangoproject.com/en/4.2/ref/settings/#databases
+# try reading ROCKY_DB_DSN from environment, if not set fallback to old environment variables
+try:
+    POSTGRES_DB = env.db("ROCKY_DB_DSN")
+    POSTGRES_DB["ENGINE"] = "django.db.backends.postgresql_psycopg2"
+except ImproperlyConfigured:
+    POSTGRES_DB = {
         "ENGINE": "django.db.backends.postgresql_psycopg2",
```

Comment:
The DB backend has been named `django.db.backends.postgresql` since Django 1.9 so we should probably use that...

Reply:
I didn't change it. But `django.db.backends.postgresql` works and should probably be used indeed